README.md

```diff
@@ -22,6 +22,19 @@ cd jr
 make release.linux -j
 ```
 
+This is producing an executable named "jerry" in "build/bin/release.linux/".
+
+### Hello world
+
+Once the executable "jerry" is built, you can try to launch the "hello.js" of the tests directory:
```

Comment:
I prefer backticks over quotes, since they stand out better when rendered. But that's not a strong opinion.